packages/plugins/ui-theme/package.json

```diff
@@ -24,38 +24,36 @@
     "@material-ui/core": "4.11.4",
     "@material-ui/icons": "4.11.2",
     "@material-ui/styles": "4.11.4",
-    "@testing-library/dom": "7.31.2",
+    "@testing-library/dom": "8.2.0",
     "@testing-library/jest-dom": "5.14.1",
-    "@testing-library/react": "11.2.7",
+    "@testing-library/react": "12.0.0",
     "@verdaccio/node-api": "workspace:6.0.0-6-next.17",
     "autosuggest-highlight": "3.1.1",
     "babel-loader": "8.2.2",
     "babel-plugin-dynamic-import-node": "2.3.3",
     "babel-plugin-emotion": "10.0.33",
     "bundlesize": "0.18.1",
-    "css-loader": "5.2.1",
+    "css-loader": "6.2.0",
     "dayjs": "1.10.6",
     "emotion": "10.0.27",
     "emotion-theming": "10.0.27",
-    "file-loader": "5.1.0",
     "friendly-errors-webpack-plugin": "1.7.0",
     "github-markdown-css": "4.0.0",
     "harmony-reflect": "1.6.2",
-    "history": "^4.10.1",
+    "history": "4.10.1",
     "html-webpack-plugin": "5.3.2",
-    "i18next": "^19.9.2",
+    "i18next": "20.6.0",
     "in-publish": "2.0.1",
     "js-base64": "3.6.1",
-    "js-yaml": "3.14.0",
-    "lint-staged": "9.5.0",
+    "js-yaml": "3.14.1",
     "localstorage-memory": "1.0.3",
     "lodash": "4.17.21",
-    "mini-css-extract-plugin": "^1.6.0",
+    "mini-css-extract-plugin": "2.2.2",
     "mutationobserver-shim": "0.3.7",
     "node-mocks-http": "1.10.1",
     "normalize.css": "8.0.1",
     "optimize-css-assets-webpack-plugin": "^5.0.8",
-    "ora": "4.0.4",
+    "ora": "6.0.0",
```

Comment:
We cannot move to `v6`, locally fails `packages/plugins/ui-theme start: Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: `. Fails on start the webpack dev server
https://github.com/sindresorhus/ora/releases/tag/v6.0.0

to reproduce:
```
pnpm build
pnpm start
```

If you find the way to solve it, cool, otherwise we must rollback to v5 at least.

Reply:
Downgrade `ora` to `v5`. Ready to review & go again.